platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HTTPRequest.java

```diff
@@ -56,13 +56,14 @@ private HTTPRequest(long nativePtr, String resourceUrl, String etag, String modi
     mNativePtr = nativePtr;
 
     try {
-      // Don't try a request if we aren't connected
-      if (!Mapbox.isConnected()) {
+      HttpUrl httpUrl = HttpUrl.parse(resourceUrl);
```

Comment:
Mmrm this code wasn't in the original PR, why was it introduced?

Reply:
I triple checked it and the code in https://github.com/mapbox/mapbox-gl-native/blob/3f5c1f12b03ecc266b1a1501df3d797efade0397/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HTTPRequest.java#L58-L65 vs https://github.com/rjuszczyk/mapbox-gl-native/blob/b113257bf5cf942029edf612313560e0a1c9bf04/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HTTPRequest.java#L58-L65 is the same except for formatting ( I rebased and fixed-up my formatting commit).

Reply:
@tobrun Thanks for checking, it must be a `diff` artifact.